src/helpers/mimetypes.js

```diff
@@ -0,0 +1,3 @@
+export default {
```

Comment:
Avoid generic names like `helpers`. They are ambiguous. `support` might not sound super meaningful, but in the software design over the years, _supporting data_ has been put into a directory called `support` by convention. So...at least there's a global convention behind it. The ideas is: as long as it doesn't do anything, but only provides some data, it's a _support_ module.
--> `src/support/mimetypes`